packages/react-native-jigsaw/src/components/Elevation.tsx

```diff
@@ -15,16 +16,27 @@ type Props = {
   theme: typeof themeT;
 } & ViewProps;
 
-const Elevation: React.FC<Props> = ({ style, theme, ...rest }) => {
-  const flattenedStyles = StyleSheet.flatten(style) || {};
-  const { elevation } = flattenedStyles;
+const Elevation: React.FC<Props> = ({ style, theme, children, ...rest }) => {
+  const { elevation = 4, borderRadius: radius } = (StyleSheet.flatten(style) ||
+    {}) as ViewStyle;
```

Comment:
why cast?

Reply:
i copied this (back) to the original implementation https://github.com/callstack/react-native-paper/blob/main/src/components/Surface.tsx#L63

we've had to do this before though so the answer is react native and typescript still don't play great together

Reply:
out of curiosity i removed the cast to see if it's been fixed and it has been, so in this case i removed it